docs/atlascli/command/atlas-dbusers-list.txt

```diff
@@ -37,6 +37,10 @@ Options
      - Type
      - Required
      - Description
+   * - -c, --compact
```

Comment:
I might be missing context. Did we agree for this flag to be present in docs (not hidden?)

Reply:
Jakub flagged that he wanted a note in --help for affected commands, which for me implies that it should be present in docs

> Add note to --help for affected commands that provide a short explanation for the relevant (affected) CLI versions